Jenkinsfile

```diff
@@ -12,17 +12,15 @@ pipeline {
     stages {
         stage('main') {
             steps {
-                sh 'mvn -B clean verify'
+                sh 'mvn -B clean install archetype:integration-test'
```

Comment:
Or could go back to `verify` if there were a plugin execution for `archetype:integration-test` in `pom.xml`.